src/components/ha-sidebar.ts

```diff
@@ -202,198 +202,227 @@ class HaSidebar extends LitElement {
   private _sortable?;
 
   protected render() {
-    const hass = this.hass;
-
-    if (!hass) {
+    if (!this.hass) {
       return html``;
     }
 
+    return html`
+      ${this.renderHeader()} ${this.renderItems()} ${this.renderDivider()}
+      ${this.renderNotifications()} ${this.renderUserItem()}
```

Comment:
```ts
      ${this.renderHeader()}
      ${this.renderPanels()}
      ${this.renderDivider()}
      ${this.renderNotificationItem()}
      ${this.renderUserItem()}
```

Reply:
It auto-prettifies to this :( :( 

I can work around it by pulling the wrapping <div>s out of those methods and surrounding the call, but that doesn't feel right to me -- a `renderX()` should be fully responsible for rendering its content.

Reply:
Disabled prettier in two places.